Makefile

```diff
@@ -408,6 +408,7 @@ ${ISTIO_OUT}/archive: istioctl-all LICENSE README.md install/updateVersion.sh re
 	cp LICENSE ${ISTIO_OUT}/archive
 	cp README.md ${ISTIO_OUT}/archive
 	cp -r tools ${ISTIO_OUT}/archive
+	go run tools/license/get_dep_licenses.go > LICENSES.txt
```

Comment:
Thanks for working on this :+1: 
As things stand this will succeed even if (say) licenses are missing. Right now nothing is missing so it should succeed but I'd recommend:
1. Add --check mode to line before and/or
2. change the code to return error for missing or illegal licenses rather than just logging error.

Reply:
@ostromart I am just erroring out if no license is found. not adding check now because that is more involved